scripts/oligomap_output_to_sam_nh_filtered.py

```diff
@@ -307,9 +296,9 @@ def get_sam_fields(aln: list[str]) -> Fields:
     return fields
 
 
-def eval_aln(outfile: Path, nhfilter: int, d: Dict[str, list],
-             minErr_nh: Dict[str, list], fields: Fields) -> None:
-    """Evaluate an alignment to add, discard or write it in the output file.
+def eval_aln(nhfilter: int, d: Dict[str, list], minErr_nh: Dict[str, list],
```

Comment:
Python conventions for variable names are to use `snake_case` consistently, not `CamelCase` or a mix of these. So better `min_err_nh` instead of `minErr_nh`.